internal/dao/stubs/stub_errors.go

```diff
@@ -25,3 +28,16 @@ func NewCreateError(ctx context.Context, stubName string) dao.Error {
 		Context: ctx,
 	}
 }
+
+func newValidationError(ctx context.Context, stubName string, model interface{}, validationErr validator.ValidationErrors) dao.ValidationError {
+	errors := []string{fmt.Sprintf("Validation of %s failed: ", stubName)}
+	for _, ve := range validationErr {
+		errors = append(errors, ve.Error())
+	}
+	msg := strings.Join(errors, ", ")
+
+	if logger := ctxval.Logger(ctx); logger != nil {
+		logger.Info().Msg(msg)
+	}
+	return dao.ValidationError{Context: ctx, Message: msg, Err: validationErr, Model: model}
+}
```

Comment:
I see it is identical to the function in sqlx_errors.go. 
I guess you prefer to repeat code while having a logical separation?

Reply:
yeah, I guess I was lazy, because this gets the stubName as a string, but we can improve that and make the Daos adhere the name for errors interface :)

Reply:
Done :+1: